src/components/EggheadPlayer/players/Bitmovin.js

```diff
@@ -105,11 +113,26 @@ export default class Bitmovin extends Base {
       },
       events: {
         playbackspeedchanged: (e) => {
+          track('set played speed', {
+            speed: e.to,
+          })
           onPlaybackRateChange(e.to)
         },
         volumechanged: (e) => {
           onVolumeChange(e.targetVolume)
         },
+        stallstarted: () => {
```

Comment:
https://cdn.bitmovin.com/player/web/8/docs/enums/core_events.playerevent.html

this makes is really easy to do event stuff without changing a bunch of files if we don't need to